shell/platform/common/cpp/text_input_model.cc

```diff
@@ -76,7 +83,7 @@ void TextInputModel::DeleteSelected() {
   selection_extent_ = selection_base_;
 }
 
-void TextInputModel::AddCharacter(char c) {
+void TextInputModel::AddCharacter(unsigned int c) {
```

Comment:
Shouldn't this be `char16_t`? Certainly we should pick a clear character type to use at this API boundary, and since the rest of this class is char16_t-based that seems like the right choice.